internal/inbox/inbox.go

```
@@ -430,11 +505,39 @@ func (m *Manager) Update(id int, inbox imodels.Inbox) (imodels.Inbox, error) {
-	if err := m.queries.Update.Get(&updatedInbox, id, inbox.Channel, encryptedConfig, inbox.Name, inbox.From, inbox.CSATEnabled, inbox.PromptTagsOnReply, inbox.Enabled, inbox.Secret, inbox.LinkedEmailInboxID, inbox.FromNameTemplate); err != nil {
+	verificationStates := make(map[string]aliasVerificationState)
+	if current.Channel == ChannelEmail {
+		var states []aliasVerificationState
+		if err := tx.Select(&states, `SELECT email, verification_token, verification_started_at FROM inbox_email_addresses WHERE inbox_id = $1 AND kind = 'alias'`, id); err != nil {
+			return imodels.Inbox{}, m.persistenceError("fetching inbox alias verification state", err)
+		}
+		for _, state := range states {
+			verificationStates[strings.ToLower(state.Email)] = state
+		}
+	}
+	if err := tx.Stmtx(m.queries.Update).Get(&updatedInbox, id, inbox.Channel, encryptedConfig, inbox.Name, inbox.From, inbox.CSATEnabled, inbox.PromptTagsOnReply, inbox.Enabled, inbox.Secret, inbox.LinkedEmailInboxID, inbox.FromNameTemplate); err != nil {
```

golangci-lint (2.13.2)

[error] Rows/Stmt/NamedStmt was not closed

(sqlclosecheck)